include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -343,8 +382,42 @@ struct pairwise_alignment_edit_distance_unbanded
     bool large_patterns();
 
 public:
+
+    /*!\brief Generic invocable interface.
+     * \param[in,out] res The alignment result to fill.
+     * \returns A reference to the filled alignment result.
+     */
+    template <typename result_type>
+    result_type & operator()(result_type & res)
+    {
+        run();
+        if constexpr (std::tuple_size_v<result_type> == 2)
+        {
+            get<align_result_key::score>(res) = score();
+        }
+        else if constexpr (std::tuple_size_v<result_type> == 3)
+        {
+            throw std::invalid_argument{"The current output setting is not supported!"};
+        }
+        else if constexpr (std::tuple_size_v<result_type> == 4)
+        {
+            throw std::invalid_argument{"The current output setting is not supported!"};
+        }
+        else if constexpr (std::tuple_size_v<result_type> == 5)
+        {
+            //TODO:
+            auto trace = alignment_trace(database, query, trace_matrix());
+            ranges::copy(std::get<0>(trace), ranges::back_inserter(std::get<0>(get<align_result_key::trace>(res))));
+            ranges::copy(std::get<1>(trace), ranges::back_inserter(std::get<1>(get<align_result_key::trace>(res))));
```

Comment:
Not sure why you copy here, because trace is already a sequence which satisfies `aligned_sequence_concept`.